integreat_cms/cms/models/languages/language.py

```diff
@@ -91,6 +91,14 @@ class Language(AbstractBaseModel):
             _("This flag is used in the language switcher."),
         ),
     )
+    language_color = models.CharField(
+        choices=language_color.COLORS,
+        max_length=7,
+        verbose_name=_("language color"),
+        blank=True,
+        null=True,
+        help_text=_("This is used to represent color label of the language"),
+    )
```

Comment:
It would be more nice if we could see which color is assigned to each language in the language list of network management and region language trees. Adding a new column in the templates will achieve it :smile_cat: 

We can probably add a new field for color selection in the language form (I suggest in the box " Extended Settings") (not the language tree form of regions, but in the network management), so it's possible to change color assignment in GUI without changing the code itsself.